tests/samplers_tests/test_gp.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from _pytest.logging import LogCaptureFixture
+
+import optuna
+
+
+def test_after_convergence(caplog: LogCaptureFixture) -> None:
```

Comment:
It is overkill to create a study and calling `study.optimize` for the tests of the internal `_gp` module. To limit the range of impact, unit tests should be written with the minimum necessary interfaces. How about just creating the `np.ndarray` and give it to `local_search_mixed`?